packages/core/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,56 @@
+import { Component, Element, h, Host, Prop } from '@stencil/core'
+
+@Component({
+  tag: 'atom-meter',
+  styleUrl: 'meter.scss',
+  shadow: true,
+})
+export class AtomMeter {
+  @Prop() type?: 'success' | 'neutral' | 'warning' | 'danger' = 'neutral'
+  @Prop() size?: 'small' | 'large' = 'large'
+  @Prop() title: string
+  @Prop() centerTitle?: boolean = false
```

Comment:
suggestion: maybe `hasCenterTitle`? (we don't have defined a pattern for it 😢 )

Reply:
Nice observation!

We generally use "has", I think it makes sense!

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/9c6099420e10ce77c49315d39f9da3221078d3c5